src/Compiler/Driver/CompilerDiagnostics.fs

```diff
@@ -387,6 +387,7 @@ type PhasedDiagnostic with
         | 3390 -> false // xmlDocBadlyFormed - off by default
         | 3395 -> false // tcImplicitConversionUsedForMethodArg - off by default
         | 3559 -> false // typrelNeverRefinedAwayFromTop - off by default
+        | 3560 -> false // tcCopyAndUpdateRecordChangesAllFields - off by default
```

Comment:
If this is the final solution, it might be good to change how the warning is raised too -`ErrorEnabledWithLanguageFeature` is now misleading..

Reply:
Not final, quickest one to not ship it.